client/cody-ui/src/chat/inputContext/ChatInputContext.module.css

```
@@ -46,3 +46,12 @@
     text-overflow: ellipsis;
     overflow: hidden;
 }
+
+.fail {
+    display: inline-flex;
+    align-items: center;
+    gap: 0.25rem;
+    overflow: hidden;
+
+    color: red;
```

Let's use this theme specific color.

```suggestion
    color: var(--vscode-errorForeground);
```

---

@philipp-spiess would the theme color work on the web UI?

---

added fallback `var(--vscode-errorForeground, red);`